package.json

```diff
@@ -46,6 +46,7 @@
     "@angular-devkit/schematics": "~0.0.51",
     "@schematics/angular": "~0.1.16",
     "autoprefixer": "^7.2.3",
+    "builtin-modules": "2.0.0",
```

Comment:
This should be a dev dependency

Reply:
Thanks @clydin.  I agree and will make the change.